WordPress/src/main/res/values/strings.xml

```diff
@@ -940,9 +940,9 @@
     <string name="editor_page_title_placeholder">Page Title</string>
     <string name="editor_content_placeholder">Share your story here…</string>
     <string name="editor_post_saved_locally">The post was saved on your device</string>
-    <string name="editor_post_saved_locally_not_published">The post was saved on your device, but wasn\'t published.</string>
+    <string name="editor_post_saved_locally_not_published">The post saved locally only. Consider publishing it.</string>
```

Comment:
Hmm, this one seems weird grammatically. Perhaps we can lose the `The` from the string start. Thanks!